PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
 ##### SUMMARY
-<!--- Describe the change below, including rationale and design decisions -->
+
+<!--- Describe the change below, including rationale and design decisions.
+The approvers and mergers shouldn't have to interpret and guess by jumping right to the code. Context helps. -->
+
+<!-- Why this change is going in? -->
+
+<!-- If applicable, what catalog item is using it? What agnosticV config is using it? -->
```

Comment:
We already kind of ask for this later on in the template, so we should get rid of that.